Src/ILGPU/Runtime/ArrayViewExtensions.Generated.tt

```diff
@@ -700,172 +700,6 @@ namespace ILGPU.Runtime
 
         #endregion
 
-        #region Copy to/from Page Locked
```

Comment:
Question: Given our attempts to maintain backwards compatibility, I was reconsidering these changes. Should we just keep all functions and mark the as `obsolete`?

Reply:
Re-added methods and marked as `[Obsolete]`.